ptf/Dockerfile

```
@@ -45,8 +45,9 @@ ARG TREX_LIBS
 
 ENV TREX_SCRIPT_DIR=/trex-core-${TREX_VER}/scripts
 
-RUN wget https://github.com/stratum/trex-core/archive/${TREX_VER}.zip && \
-    unzip -qq ${TREX_VER}.zip && \
+RUN apk update && apk add --no-cache -U wget
+RUN wget https://github.com/stratum/trex-core/archive/refs/heads/v${TREX_VER}.zip && \
```

Why do you have to make this change? Why not just something like this:
```suggestion
RUN wget https://github.com/stratum/trex-core/archive/v${TREX_VER}.zip && \
```

---

done